app/src/main/kotlin/com/wire/android/ui/home/conversations/mention/MemberItemToMention.kt

```diff
@@ -82,7 +82,7 @@ fun PreviewMemberItemToMention() {
         name = "name",
         label = "handle",
         membership = Membership.Federated,
-        searchQuery = "search",
```

Comment:
I know its just a preview but maybe we keep the searchQuery as it was before?

Reply:
previous version of preview was showing nothing, cause there is no "search" in username, so it hightlight nothing. "na" on the other hand highlight at least something in preview :)